src/main/java/by/onliner/page/pages/MainPage.java

```diff
@@ -0,0 +1,54 @@
+package by.onliner.page.pages;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.page.AbstractPage;
+import by.onliner.page.components.Menu;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+import org.openqa.selenium.support.PageFactory;
+import java.util.List;
+import static by.onliner.constants.OnlinerConstants.HOST;
+import static by.onliner.util.ListUtil.getRandomWebElementFromList;
+
+public class MainPage extends AbstractPage {
+
+    private final int maxListSizeOnPage = 7;
+
+    @FindBys({
```

Comment:
In what cases FindBys is used? are you sure you are using it right?

Reply:
Renamed to FindBy